src/pages/Home/Home.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import Form from 'react-bootstrap/Form';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import Cards from '../../components/Card/Cards';
+import { allCoins } from '../../redux/home-reducer';
+
+const Home = () => {
+  const [search, setSearch] = useState('');
+  const handleSearch = (e) => {
+    setSearch(e.target.value);
+  };
+  const Coins = useSelector((state) => state.coin);
```

Comment:
- [ ] Kindly improve the design of your Hope page as well to match the original one (Typography: font face, size, background, color, weight...)👇🏻 

Your Implementation | Original Design
:---:|:---:
![image](https://user-images.githubusercontent.com/79058364/218008640-529056c0-6887-45e9-ba3d-0d9e4dab8d5f.png)| ![Screenshot 2023-02-09 at 10 15 21 (2)](https://user-images.githubusercontent.com/64774936/217769278-651d6695-ed6d-4197-9584-53be3452a033.png)